From 99aa50e4972f037e8f807a938e8062b54f2394eb Mon Sep 17 00:00:00 2001
From: Tomas Winkler <tomas.winkler@intel.com>
Date: Sun, 16 Mar 2014 14:35:57 +0200
Subject: [PATCH 329/441] mei: amthif: fix checkpatch error

ERROR: else should follow close brace '}'
+	}
+	else {

Change-Id: I4ba3ff0ff9aa596ba2ce0a81cd8e8aad728770b3
Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
(cherry picked from commit c10f97273cd657d01dd8f4358d80b8c919d09cfb)

Signed-off-by: Nagaraj S <sivasankaranx.nagaraj@intel.com>
---
 drivers/misc/mei/amthif.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/misc/mei/amthif.c
+++ b/drivers/misc/mei/amthif.c
@@ -247,8 +247,7 @@ int mei_amthif_read(struct mei_device *d
 	if (copy_to_user(ubuf, cb->response_buffer.data + *offset, length)) {
 		dev_dbg(&dev->pdev->dev, "failed to copy data to userland\n");
 		rets = -EFAULT;
-	}
-	else {
+	} else {
 		rets = length;
 		if ((*offset + length) < cb->buf_idx) {
 			*offset += length;
